src/Reducer.js

```diff
@@ -48,6 +48,11 @@ function inject(state, action, props, scenes) {
             case POP_ACTION:
                 assert(!state.tabs, "pop() operation cannot be run on tab bar (tabs=true)")
                 return {...state, index:state.index-1, children:state.children.slice(0, -1) };
+            case POP_TO_TOP:
+                assert(state.tabs, "Parent="+state.key+" is not tab bar, popToTop action is not valid");
+                let child = state.children[state.index];
+                let newChild = {...child, ...{children: [child.children[0]], index:0}};
+                return {...state, ...{children:[...state.children.slice(0,state.index), newChild, ...state.children.slice(state.index+1)]}};
```

Comment:
Wouldnt that mean I'd have to hardcode it for each of my tabs? I have not named my tabs as in the example, seemed arbitrary? Is the naming convention important here?

This is my router code with a small set of views in there.
```js
 return <Router createReducer={reducerCreate} store={store}>
      <Scene key="tabBar" tabs={true} tabBarStyle={{backgroundColor:colors.menuBackground.h}}>
        <Scene key="overview" title="Overview" icon={TabIcon} iconString="ios-color-filter-outline" navigationBarStyle={{backgroundColor:colors.menuBackground.h}} titleStyle={{color:'white'}} renderBackButton={backButtonFunction}>
          <Scene key="groupOverview" component={GroupOverview} title="Group Overview" />
          <Scene key="roomOverview"  component={RoomOverview} onRight={onRightFunctionEdit} rightTitle="Edit" rightButtonTextStyle={{color:'white'}} />
          <Scene key="roomEdit"      component={RoomEdit} title="Edit Room" />
          <Scene key="deviceEdit"    component={DeviceEdit} title="Edit Device" />
        </Scene>
        <Scene key="settings" title="Settings" icon={TabIcon} iconString="ios-gear-outline" navigationBarStyle={{backgroundColor:colors.menuBackground.h}} titleStyle={{color:'white'}} renderBackButton={backButtonFunction}>
          <Scene key="Settings" component={SettingsOverview} title="Settings"/>
        </Scene>
      </Scene>
    </Router>;
```

Whenever I refer to specific scenes inside the tabs, the tabbar disappears.

If I have to hardcode the action key in the reducer for each of my tabs it seems a bit more sloppy than using the index of the state?